Fix variable naming bug

kreuzhofer opened the pull request:
Using "json" instead of "message" in the Main() function fixes compilation bug.

azurecla commented:
Hi __@HEDIDIN__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

tysonn commented:
@HEDIDIN - Thank you for the fix!

kreuzhofer committed:
Fix variable naming bug

Using "json" instead of "message" in the Main() function fixes compilation bug.
